src/zig_clang.cpp

```diff
@@ -1625,6 +1625,10 @@ unsigned ZigClangFunctionDecl_getAlignedAttribute(const struct ZigClangFunctionD
     return 0;
 }
 
+ZigClangQualType ZigClangParmVarDecl_getType(const struct ZigClangParmVarDecl *self) {
```

Comment:
should be named `ZigClangParmVarDecl_getOriginalType` after the function that it is calling